.travis.yml

```diff
@@ -19,11 +19,13 @@ install:
   - tar -xvf $PWD/travis-phantomjs/phantomjs-2.0.0-ubuntu-12.04.tar.bz2 -C $PWD/travis-phantomjs
   - export PATH=$PWD/travis-phantomjs:$PATH
 
+  - cp ./api/.travis.env ./api/.env
+
   - travis_retry npm install --no-spin --quiet -g gulp http-server
   - travis_retry npm install --no-spin
   - travis_retry gulp bower:install
   - travis_retry composer self-update
-  - travis_retry composer install --prefer-source --no-interaction --working-dir api
+  - travis_retry composer install --prefer-source --no-interaction --optimize-autoloader --working-dir api
```

Comment:
I've noticed that there is an issue with composer not picking up namespacing when not autoloading with optimized, any idea why this might be?